### ISSUE TITLE
Pip installs 0.1 by default.

### ISSUE THREAD

**Issue by garnertb** [lamby/django-slack#13]
I would expect to get version 4.
```
$ pip freeze | grep slack
$ pip install django-slack
Downloading/unpacking django-slack
Downloading django-slack-0.0.1.tar.gz
...
Installing collected packages: django-slack
  Running setup.py install for django-slack
Successfully installed django-slack
Cleaning up...
$ pip freeze | grep slack
django-slack==0.0.1
```

**Comment by lamby** [comment]
Can't reproduce that here:

![](http://i.imgur.com/fIsJMSx.png)

**Comment by garnertb** [comment]
Interesting, here it is again in a new virtual environment on Ubuntu 14.04 with pip
```
$ virtualenv test --no-site-packages
New python executable in test/bin/python
Installing setuptools, pip...done.
$ source test/bin/activate
$ pip --version
pip 1.5.4 from /tmp/test/local/lib/python2.7/site-packages (python 2.7)
$ pip install django-slack
Downloading/unpacking django-slack
  Downloading django-slack-0.0.1.tar.gz
  Running setup.py (path:/tmp/test/build/django-slack/setup.py) egg_info for package django-slack
    
Downloading/unpacking Django>=1.6.8 (from django-slack)
  Downloading Django-1.8.5-py2.py3-none-any.whl (6.2MB): 6.2MB downloaded
Downloading/unpacking requests (from django-slack)
  Downloading requests-2.8.1-py2.py3-none-any.whl (497kB): 497kB downloaded
Installing collected packages: django-slack, Django, requests
  Running setup.py install for django-slack
    
Successfully installed django-slack Django requests
Cleaning up...
```

Works fine on my host machine: Mac OSX:
```
$ mktmpenv
New python executable in tmp-87e5baae9493d5f3/bin/python
Installing setuptools, pip...done.
This is a temporary environment. It will be deleted when you run 'deactivate'.
(tmp-87e5baae9493d5f3)$ pip install django-slack
Downloading/unpacking django-slack
  Downloading django-slack-4.tar.gz
  Running setup.py (path:/Users/env/tmp-87e5baae9493d5f3/build/django-slack/setup.py) egg_info for package django-slack
    
Downloading/unpacking Django>=1.6.8 (from django-slack)
  Downloading Django-1.8.5-py2.py3-none-any.whl (6.2MB): 6.2MB downloaded
Downloading/unpacking requests (from django-slack)
  Downloading requests-2.8.1-py2.py3-none-any.whl (497kB): 497kB downloaded
Installing collected packages: django-slack, Django, requests
  Running setup.py install for django-slack
    
Successfully installed django-slack Django requests
Cleaning up...
(tmp-87e5baae9493d5f3)$ pip freeze
Django==1.8.5
django-slack==4
requests==2.8.1
wsgiref==0.1.2
(tmp-87e5baae9493d5f3)$ pip --version
pip 1.5.6 from /Users/env/tmp-87e5baae9493d5f3/lib/python2.7/site-packages (python 2.7)
```

**Comment by lamby** [comment]
Try ``--no-cache-dir`` or clearing your Pip cache?

**Comment by garnertb** [comment]
Couldn't find any evidence of it being cached (--no-cache-dir isn't introduced until 6.0 so that wasn't an option).  

Upon further investigation, this seems to only affect pip==1.5.4 (I didn't try <1.5.4).  You should be able to reproduce the issue with:

```
pip install pip==1.5.4
pip uninstall django-slack
pip install django-slack
```

I couldn't find the stats for number of users installing packages with pip 1.5.4 so feel free to close/fix this based on the version of pip you officially support.  Thanks for the help.

**Comment by lamby** [comment]
I added ``--verbose`` which led to:

```
>>> from pip.util import is_prerelease
>>> is_prerelease('4')
True
```

So, specifying ``--pre`` would work right now. Also pip thinks that this version number style is invalid, so I need to change that. (And will do)

**Comment by lamby** [comment]
Just released "4.0.1". Closing - thanks!

**Comment by NejcZupec** [comment]
Good job! :)
